recipes/ca-certificates/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "2016.2.28" %}
+
+package:
+  name: ca-certificates
+  version: {{ version }}
+
+# This does not have a source because it pulls these from certifi at present.
+
+build:
+  number: 0
+  skip: true  # [win or not py35]
```

Comment:
Not sure if we need this on Windows. Thinking not based on our use in `git`. Though `curl` was going to use this *I believe* so maybe?

Also, while this requires `python` (via `certifi`), it does not need to be built for multiple Python versions as it just copies these. So, I just picked one arbitrarily.